Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -67,9 +66,7 @@ class RemoteNotificationHandler {
 
     func safeAdded(address: Address) {
         logDebug("Safe added: \(address)")
-        if authorizationStatus == nil {
```

Comment:
Can you tell more about why this was removed? Will we have the push notification permission when the safe is added?

Reply:
I removed it because I found that it is requested on every app start anyways. But you are right, before it was better 👍 
I will returen it